compliance/collection/metrics/metrics.go

```diff
@@ -300,10 +315,20 @@ func ObserveNodeInventorySending(nodeName string, sendingType InventoryTransmiss
 	}).Inc()
 }
 
+// ObserveNodePackageReportTransmissions observes the metric.
+func ObserveNodePackageReportTransmissions(nodeName string, sendingType InventoryTransmission, scannerVersion string) {
+	nodePackageReportTransmissions.With(prometheus.Labels{
+		"node_name":         nodeName,
+		"transmission_type": string(sendingType),
+		"scanner_version":   scannerVersion,
+	}).Inc()
+}
```

Comment:
How about having single observe function that updates both metrics?

```suggestion
func ObserveNodePackageReportTransmissions(nodeName string, sendingType InventoryTransmission, scannerVersion string) {	
	nodePackageReportTransmissions.With(prometheus.Labels{
		"node_name":         nodeName,
		"transmission_type": string(sendingType),
		"scanner_version":   scannerVersion,
	}).Inc()
	// Recorded additionally in the old metric for backwards compatibility.
	// Remove the block below after dropping Stackrox Scanner (aka Scanner v2).
	inventoryTransmissions.With(prometheus.Labels{
		"node_name":         nodeName,
		"transmission_type": string(sendingType),
		"scanner_version":   scannerVersion,
	}).Inc()
}
```